frontend/src/app/components/Leaderboards/Leaderboards.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import { ListCon, FiltersBlock, FiltersSpan } from '../../../styles/blocks'
+import { LeaderboardsRow } from './LeaderboardsRow'
+import { getTopUsers } from '../../modules/users/users-selectors'
+import { LeaderboardsActions } from '../../modules/leaderboards/leaderboards-actions'
+import * as Filters from '../../const/leaderboards-filters'
+
+const LeaderboardsComponent = (
+  { topUsers, filters, maxItems, updateCriteria, updateOrder, updateTimespan }
+) =>
+  (
+  <>
+    <FiltersBlock>
+      {Object.entries(Filters.criteriaTypes).map(([key, value]) =>
+        <FiltersSpan key={key} className={value === filters.criteria ? 'active-filter' : ''}
+          onClick={() => updateCriteria(value)}>{key}</FiltersSpan>,
+      )}
+
+      {Object.entries(Filters.orderTypes).map(([key, value]) =>
+        <FiltersSpan key={key} className={value === filters.order ? 'active-filter' : ''}
+          onClick={() => updateOrder(value)}>{value}</FiltersSpan>,
+      )}
+
+      {Object.entries(Filters.timespanTypes).map(([key, value]) =>
+        <FiltersSpan key={key} className={value === filters.timespan ? 'active-filter' : ''}
+          onClick={() => updateTimespan(value)}>{key.split(/(?=[A-Z0-9])/).join(' ')}
+        </FiltersSpan>,
+      )}
+    </FiltersBlock>
+
+    <ListCon className="topPlayers">
+      {topUsers.slice(0, maxItems).map((user, index) =>
+        <LeaderboardsRow key={user.id} user={user} position={index}
+          points={getUserPoints(user, filters.criteria)} />,
+      )}
+    </ListCon>
+  </>
+  )
+
+const getUserPoints = (user, criteria) => {
+  switch (criteria) {
+    case Filters.criteriaTypes.Wins:
```

Comment:
why do you capitalize filters?

Reply:
To distinguish imported module with constants from filter object parsed out from state